stdlib/asm/crypto/stark/random_coin.masm

```diff
@@ -181,13 +183,21 @@ export.init_seed
     push.0x01 # lower half of the modulus
     push.0xffffffff # upper half of the modulus
     ## field extension and FRI parameters
-    ## field extension degree || FRI folding factor || FRI remainder polynomial max degree
-    push.0x020407
+    ## field extension degree || FRI folding factor || FRI remainder polynomial max degree || blowup factor
+    push.0x02040708
+    # => [PROOF_OPTIONS, MODULUS1, MODULUS0, TRACE_INFO, trace_length, num_queries, blowup, grinding, ...]
```

Comment:
`MODULUS1` and `MODULUS0` are single elements, right? If so, these should be `modulus1, modulus0`.

Reply:
good point, fixed